fr-FR.lproj/Localizable.strings

```diff
@@ -0,0 +1,369 @@
+/* 
+  Localizable.strings
+  Undecimus
+
+  Created by Brieuc Caillot on 8/26/19.
+  Copyright © 2019 Pwn20wnd. All rights reserved.
+*/
+
+"Bundled Resources version missing." = "La version des ressources groupées est manquante."
+"Unsupported" = "N'est pas prise en charge"
+"Restore RootFS" = "Restaurer RootFS"
+"Ready to restore RootFS" = "Prêt à restaurer RootFS"
+"Re-Jailbreak" = "Re-Jailbreak"
+"Ready to re-jailbreak" = "Ready to re-jailbreak"
+"Jailbreak" = "Jailbreak"
+"Ready to jailbreak" = "Prêt à jailbreak"
+"Error" = "Erreur"
+"Bundled Resources version is missing. This build is invalid." = "La version des ressources groupées est manquante. Le build est invalide."
+"Days" = "Jours"
+"Invalid Entry" = "Entrée non valide"
+"The boot nonce entered could not be parsed" = "Le méthode de signature au démarrage entrée n'a pas pu être analysé"
+"OK" = "OK"
+"The device will be restarted." = "L'appareil va redémarrer."
+"Copy boot nonce?" = "Copier la méthode de signature au démarrage ?"
+"Would you like to copy nonce generator to clipboard?" = "Voulez-vous copier la méthode de signature au démarrage dans le presse papier ?"
```

Comment:
Voulez-vous copier le générateur nonce (difficile car ce sont des termes techniques..)

Reply:
tu ne devrais pas traduire "boot nonce"

Reply:
"OK" = "OK"
"The device will be restarted." = "L'appareil va redémarrer."
"Copy boot nonce?" = "Copier boot nonce ?"
"Would you like to copy nonce generator to clipboard?" = "Voulez-vous copier nonce generator dans le presse papier ?"


oui vic a raison traduit que se qui et difficile pour les debutant mais nonce boot reste le nonce boot car si il rencontre un probleme il ne seront pas ou chercher